frontend/src/components/search/components/common/video-tag/styles.module.scss

```diff
@@ -0,0 +1,24 @@
+@import '../../../../../assets/css/breakpoint.scss';
```

Comment:
use absolute imports

In your solution it would be a problem to move component to another file

Apply for this comment for each files you have